.github/workflows/release-automation.yaml

```diff
@@ -3,7 +3,7 @@ on:
   workflow_dispatch:
   workflow_call:
 env:
-  GO_VERSION: '1.22.7'
+  GO_VERSION: '1.23.0'
```

Comment:
Should we be using 1.23.7?

Reply:
Yeah I think it would be good to go ahead and use the latest 1.23 across all CI workflows while you got the hood open.

Reply:
Updated to golang 1.23.7